src/api/public/apidocs-new/paths/source_project_name_cmd_remove_flag.yaml

```diff
@@ -0,0 +1,61 @@
+post:
+  summary: Delete a flag for the project.
+  description: |
+    For the project, you can delete a flag with the provided flag name, repository and architecture.
+    You can only delete the flag if you have the permissions to do so.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: query
+      name: flag
+      required: true
+      schema:
+        type: string
+      description: Name of the flag to be deleted
+      example: access
+    - in: query
+      name: repository
+      required: false
+      schema:
+        type: string
+      description: Repository for which the flag is set
+      example: openSUSE_Tumbleweed
+    - in: query
+      name: arch
+      required: false
+      schema:
+        type: string
+      description: Architecture for which the flag is set
+      example: x86_64
+  responses:
```

Comment:
Missing 404 response. Adding this should be enough:
```
    '404':
      $ref: '../components/responses/unknown_project.yaml'
```

The same happens in the `set_flag` command.